src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -359,13 +374,13 @@ def register_dataset_udf(
             ingest_list_node,
             config=config,
             verbose=verbose,
-            acn=access_credentials_name,
+            acn=acn,
             namespace=namespace,
             name=f"{dag_name} registrator ",
             expand_node_output=ingest_list_node,
             resources=DEFAULT_RESOURCES if resources is None else resources,
             image_name=DEFAULT_IMG_NAME,
-            **kwargs,
+            access_credentials_name=acn**kwargs,
```

Comment:
Extremely important forgotten comma here (unless you want to calculate <var>acn</var><sup><var>kwargs</var></sup>)